### PR TITLE
rwd: added chapter 11

### PR DESCRIPTION
**Pull request by LordRafa**

### CODE REVIEW

File: chapter11/rwd/ex2-11.pl [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+use warnings;
+use DateTime;
+
+my $today = DateTime->today( );
+my $out_date = $today->subtract_datetime( DateTime->new( year => $ARGV[0], month => $ARGV[1], day => $ARGV[2] ) );
+
+print $out_date->years() . " years, " . $out_date->months() . " months, and " . $out_date->days() . " days\n";
+
+exit 1;
+
```
Comment (pratyp) [inline chapter11/rwd/ex2-11.pl:13]: <code>exit 1</code> is for error cases, probably not the best use of it.

File: chapter11/rwd/ex1-11.pl [hunk 2]
```diff
@@ -0,0 +1,9 @@
+#!/urs/bin/perl
```
Comment (thatguy59) [inline chapter11/rwd/ex1-11.pl:1]: urs? Doesn't this stop it from running?

File: chapter11/rwd/ex2-11.pl (hunk 1 above)
Comment (alexdeas) [inline chapter11/rwd/ex2-11.pl:13]: Until you know more about exit codes, try to avoid exiting programs like this.

Comment (ardavey) [inline chapter11/rwd/ex2-11.pl:13]: We discussed this last week, and why it was a bad habit that needs to be broken.

**Pull request merged**